assets/runtime/functions

```diff
@@ -209,7 +221,7 @@ redmine_check_database_connection() {
 }
 
 redmine_configure_database() {
-  echo -n "Configuring redmine::database"
+  echo "Configuring redmine::database"
```

Comment:
Pretty sure this was intentional so the database check could print '.' while it waited for the database to start.